test/cql-pytest/random_util.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2020-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+##################################################################
+
+# Various utility functions which are useful for multiple tests.
+# Note that fixtures aren't here - they are in conftest.py.
+
+# Various utility functions to generate random elements. Using
+# randomized elements allow to have great coverage with time.
+#
+# Use fixture `randomizer_seed` present in conftest.py, to print
+# seed used for generation. If a test fails, this seed can be used to
+# reproduce the test.
+
+import random
+from contextlib import contextmanager
+from util import new_type, unique_name, new_test_table, new_test_keyspace
+from pytest import fixture
+
+# When generating random type if depth > max_depth
+# then force native_type
+max_depth = 2
+# How many types can be in collection/tuple/UDT
+max_types = 5
+# How many rows can be in partition key
+max_pk = 3
+# How many rows can be in clustering key
+max_ck = 5
+# How many regular rows cam be in table
+max_regular = 4
+# How many static rows cam be in table
+max_static = 3
+# Chance of counter table instead of normal one
+counter_table_chance = 0.1
+
+# A utility function for obtaining random native type
+def get_random_native_type():
+    types = ["text", "bigint", "boolean", "decimal", "double", "float", "int"]
+    return random.choice(types)
+
+# A utility function for obtaining random collection type.
+# `udts` argument represents user-defined types that can be used for
+# creating the collection type. If `frozen` is true, then the function will
+# return `frozen<collection_type>`
+def get_random_collection_type(cql, keyspace, udts=[], frozen=False, depth=0):
+    collection_type = random.randrange(3)
+
+    return_type = ""
```

Comment:
How can this default value ever be used below?

Reply:
Removed, it was done to initialize the variable, I forgot this is not required in python